tests/remote/test_firefox.py

```diff
@@ -101,6 +101,7 @@ def test_daily_scraper(tmpdir, args):
     mozdownload.DailyScraper(destination=tmpdir, **args)
 
 
+@pytest.mark.skip(reason="Fails with network timeout for some parameters")
```

Comment:
For mozilla-central tests I do not really expect such a failure. Do you have more details like a formerly failing CI job?